cli/src/commands/init/index.ts

```diff
@@ -233,8 +233,11 @@ export default class Init extends BaseCommand {
 
     if (!apiKey) {
       apiKey = await createAPIKeyThroughWebUI();
+      this.apiKeyLocation = 'new';
```

Comment:
is this change intended? I don't see any other reference to `apiKeyLocation`

Reply:
It's intended. It is used to print this error message: https://github.com/xataio/client-ts/blob/697d75938be15e18d05c3a77f87d6ba364c5859b/cli/src/base.ts#L184-L189

For some reason it was missing.